### PR TITLE
Bug 1573957 Added note clarifying ES sizing

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1573957

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -815,10 +815,17 @@ requires creating a deployment configuration for each Elasticsearch cluster
 node.
 
 The simplest way to change the scale of Elasticsearch is to modify the inventory
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/change the scale of/scale up

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -815,10 +815,17 @@ requires creating a deployment configuration for each Elasticsearch cluster
...
-host file a re-run the logging playbook as desribed previously. Assuming you
+host file and re-run the logging playbook as described previously. Assuming you
 have supplied persistent storage for the deployment, this should not be
 disruptive.
 
+[NOTE]
+====
+Resizing an Elasticsearch cluster using the logging playbook is only possible when
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: I'd remove "(scaled up)" here and move it up.

I'd also rewrite the note to make more active, but ¯\_(ツ)_/¯:
You can use the logging playbook to only scale up an Elasticsearch cluster. The 
new `openshift_logging_es_cluster_size` value must be greater than the current number
of Elasticsearch nodes in the cluster.

**Comment by ahardin-rh** [comment]
LGTM!

**Review by kalexand-rh — COMMENTED** [review]
I think it works, but I have a suggestion, if you want it. :)

**Pull request merged**